index.html

```diff
@@ -1,657 +1,1295 @@
+                                </div>
+                              </div>
+                            </div>
+                          </div>
                         </div>
-                    </a>
-                </div>
-                <div class="button resource-btn">
-                    <a href="" id="" class="btn-link" data-cmp-clickable="" data-cmp-data-layer="">
-                        <span class="btn-text">All Resources</span>
-                    </a>
+                      </div>
+                    </div>
+
+                    <div class="button aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                      <a id="button-dbb21d03a5" class="cmp-button" href="/content/leaderimpact/lang/en2/resources.html">
+                        <span class="cmp-button__text">All Resources</span>
+                      </a>
+                    </div>
+                  </div>
                 </div>
+              </div>
             </div>
-        </section>
-        <section class="impact-story">
-            <div class="container">
-                <div class="stories-wrap">
-                    <div class="title">
-                        <div data-cmp-data-layer="" id="" class="title-wrap">
-                            <h2>Impact Stories</h2>
-                        </div>
+          </div>
+        </div>
+
+        <div class="container responsivegrid impact-story aem-GridColumn aem-GridColumn--default--12 layout-grid">
+          <div id="impact-container" class="cmp-container">
+            <div class="aem-Grid aem-Grid--12 aem-Grid--default--12 wrap-impact-story">
+              <div class="container responsivegrid stories-wrap aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                <div id="container-112257af9a" class="cmp-container">
+                  <div class="aem-Grid aem-Grid--12 aem-Grid--default--12 ">
+                    <div class="title aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                      <div id="title-63091b8133" class="cmp-title">
+                        <h2 class="cmp-title__text">Impact Stories</h2>
+                      </div>
                     </div>
-                    <div class="stories-item-wrap">
-                        <a href="#story1" class="story-item">
-                            <div class="img">
-                                <img srcset="./assets/story1.png" src="./assets/story1.png" loading="lazy" class="post-img" itemprop="contentUrl"  alt="" title="">
+
+                    <div class="container responsivegrid aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                      <div id="container-9aefcfdaa3" class="cmp-container">
+                        <div id="stories-item-wrap" class="aem-Grid aem-Grid--12 aem-Grid--default--12 ">
+                          <div class="teaser story-item aem-GridColumn aem-GridColumn--default--12 layout-grid" data-val="#tabs-31c70c9208-item-b17065cfe5-tabpanel">

```

Comment:
We have no way to set `data-val` properties here. Your code relies upon it but we cannot set it. See [here](https://github.com/CruGlobal/leader-impact-review/blob/aem-restyle/aem/aem-home.html#L747) for the actual HTML.

Reply:
This still needs to be resolved somehow. Maybe we need to add a class and target that instead.